src/components/SuggestedThings/Box.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import '../../../src/assets/css/Box.css';
+import suggestedThings from './suggestedThings';
+
+// This functions returns the individual suggested item image and name of a particular weather type.
+function Item(props) {
+    return (
+        <div key={props.name} className="flex suggested-thing">
+            <img className="icons" src={props.img} alt={props.alt} width="50" height="50"></img>
```

Comment:
`<img> ` is a self-closing tag in React, so instead of closing it with `</img>`, use `<img .... />`

Reply:
I have used <img../> tag now.